cartoframes/data/dataset.py

```diff
@@ -1,15 +1,15 @@
-import time
 import pandas as pd
-import binascii as ba
 
 from tqdm import tqdm
 from warnings import warn
 
-from carto.exceptions import CartoException, CartoRateLimitException
+from carto.exceptions import CartoException
 
+from .utils import decode_geometry, compute_query, compute_geodataframe, get_columns, \
```

Comment:
The max length is 120. You can put the constant in the same line

Reply:
I'm not used to 120, but I can change it